bindings/matrix-sdk-ffi/src/timeline/mod.rs

```diff
@@ -1005,7 +1005,10 @@ impl TimelineItem {
 #[derive(Clone, uniffi::Enum)]
 pub enum EventSendState {
     /// The local event has not been sent yet.
-    NotSentYet,
+    NotSentYet {
+        /// When the send was first attempted.
```

Comment:
Call me uber-nitpicky :sweat_smile: but in most of the code comments relating to `created_at`: we're not talking about the time where it was first *sent*, but really when the request was created / the event was queued for sending. The request may be pushed while the device is offline, so the first send would happen only when the device gets back online, which could be hours/days later.

Reply:
Yes you are correct, I will update this language throughout:
```
/// When the send was first enqueued by the user.
```